comp/core/workloadmeta/collectors/internal/ecs/ecs.go

```diff
@@ -289,3 +452,42 @@ func (c *collector) getResourceTags(ctx context.Context, entity *workloadmeta.EC
 
 	return rt
 }
+
+func v1TaskToV4Task(task v1.Task) v3or4.Task {
+	result := v3or4.Task{
+		TaskARN:       task.Arn,
+		DesiredStatus: task.DesiredStatus,
+		KnownStatus:   task.KnownStatus,
+		Family:        task.Family,
+		Version:       task.Version,
+		Containers:    make([]v3or4.Container, 0, len(task.Containers)),
+	}
+
+	for _, container := range task.Containers {
+		result.Containers = append(result.Containers, v3or4.Container{
+			Name:       container.Name,
+			DockerName: container.DockerName,
+			DockerID:   container.DockerID,
+		})
+	}
+	return result
+}
+
+func jitter(s string) time.Duration {
+	h := fnv.New32a()
```

Comment:
Should we use the same `fnv.New32a()` instance for all our jitter calls, and make sure we call `Reset()`?

Reply:
https://github.com/DataDog/datadog-agent/blob/882d6e4354906f23c0124aa0d908eed0e5bcbd4a/comp/core/workloadmeta/collectors/internal/ecs/ecs.go#L487-L497